cabal.project

```diff
@@ -160,8 +160,8 @@ source-repository-package
 source-repository-package
   type: git
   location: https://github.com/input-output-hk/ouroboros-network
-  tag: 96cf17bcc6ea4ef455a19430313ce17d476d62b5
-  --sha256: 00xp605lb8qp1jhp43mg7818x8yzn2rnsd18qcy5721yj4q675nz
+  tag: c7455773628b4b853d908629a41fd7f9bfd49ea0
```

Comment:
This will have to change again once we merge the corresponding PR in `ouroboros-network` repo.

Reply:
Is this the https://github.com/input-output-hk/ouroboros-network/pull/2947 PR in question?